tools/ingest_historical_csv.py

```diff
@@ -0,0 +1,82 @@
+'''Ingests historical CSV data into DB'''
+
+import argparse
+import asyncio
+import logging
+import textwrap
+
+from aiohttp import ClientSession
+import pandas as pd
+import daiquiri
+
+daiquiri.setup(level=logging.INFO)
```

Comment:
We can read logging mode form config

Reply:
Sure, I will introduce a new switch. e.g `--log-level INFO`